addons/website_slides/controllers/main.py

```diff
@@ -1127,6 +1130,8 @@ def slides_embed(self, slide_id, page="1", **kw):
             if is_embedded:
                 request.env['slide.embed'].sudo()._add_embed_url(slide.id, referrer_url)
             values = self._get_slide_detail(slide)
+            if not values:
+                return NotFound()
```

Comment:
Shouldn't we raise instead of returning the error? I'm sure it's the case currently, but maybe it isn't so in earlier versions?
Also, why don't we simply check that the slide is not archived, as it is done in `slide_view`?